service/job/job_controller.py

```diff
@@ -29,11 +30,15 @@ def get_job_config():
 
 @job_controller.route('/jobs', methods=['GET'])
 @auth.login_required
-def job_list():
+def job_list(show_all_jobs=False):
```

Comment:
Es fehlt die Dokumentation im docstring für API-Doc. Als Beispiel schau dir
service/job/job_type_controller.py -> get_job_type_detail
an. Da gibt es auch einen GET-paramter mit Doku.
":query schema: (optional) if set to 'true', parameter schema is served"